packages/flutter/lib/src/widgets/mimic.dart

```diff
@@ -52,6 +123,15 @@ class MimicableState extends State<Mimicable> {
     return new MimicableKey._(this);
   }
 
+  MimicOverlayEntry liftToOverlay() {
+    OverlayState overlay = Overlay.of(context);
+    if (overlay == null)
+      return null;
+    MimicOverlayEntry mimic = new MimicOverlayEntry._(startMimic());
```

Comment:
call this variable something like "entry" rather than "mimic" since it isn't a mimic.